packages/crypto/src/__tests__/crypto-test.ts

```diff
@@ -6,7 +6,7 @@ import {
 import {
   getPublicKey,
   generateP256KeyPair,
-  decryptBundle,
+  decryptEmailBundle,
```

Comment:
just curious, why this renaming?

Reply:
Just for consistency. We now have decryptExportBundle so figured a default decryptBundle name for a function that is specifically intended for email bundles doesn't make much sense